src/svc.c

```diff
@@ -184,7 +184,7 @@ svc_init(svc_init_params *params)
 	if (__svc_params->ioq.thrd_max < params->ioq_thrd_max)
 		__svc_params->ioq.thrd_max = params->ioq_thrd_max;
 
-	work_pool_params.thrd_min = __svc_params->ioq.thrd_min + channels;
+	work_pool_params.thrd_min = __svc_params->ioq.thrd_min;
```

Comment:
Shouldn't this be part of other patch?

Reply:
Other patch we are avoiding thread_max < channels, that was possible only after this change and missed in first patch.